pkg/shell/hosts_dialog.jsx

```diff
@@ -475,7 +477,15 @@ class HostKey extends React.Component {
                 <ClipboardCopy isReadOnly hoverTip={_("Copy")} clickTip={_("Copied")} className="hostkey-fingerprint pf-v5-u-font-family-monospace">{fp}</ClipboardCopy>
                 <p className="hostkey-type">({key_type})</p>
                 <p>{cockpit.format(_("To verify a fingerprint, run the following on $0 while physically sitting at the machine or through a trusted network:"), this.props.host)}</p>
-                <ClipboardCopy isReadOnly hoverTip={_("Copy")} clickTip={_("Copied")} className="hostkey-verify-help-cmds pf-v5-u-font-family-monospace">ssh-keyscan -t {key_type} localhost | ssh-keygen -lf -</ClipboardCopy>
+                <ClipboardCopy isReadOnly hoverTip={_("Copy")} clickTip={_("Copied")} className="hostkey-verify-help-cmds pf-v5-u-font-family-monospace">{`ssh-keyscan -t ${key_type} localhost | ssh-keygen -lf -`}</ClipboardCopy>
+                <p>{_("Optionally, paste the result below to compare fingerprints")}</p>
+                <Form>
```

Comment:
This is a lot of duplicated code... should we refactor this into a re-usable component? That component would probably contains also the ClipboardCopy's and other existing code that is already duplicated. If you don't want to do this, that's fine as well.

However, we should avoid the duplicate `id="command-ssh-keyscan-result-paste"` attribute. Does the textarea need an `id` attribute?

Reply:
I figured `id` will make it easier to write the tests and they are essentially unique as only one of them gets instantiated, I don't see any other use apart from that.

`id` can be removed, I'll remove it after getting acquainted with the automated tests.